src/simulation/m_rhs.fpp

```diff
@@ -831,24 +801,16 @@ contains
                                                  qR_prim, &
                                                  q_prim_qp, &
                                                  dq_prim_dx_qp, dq_prim_dy_qp, dq_prim_dz_qp, &
-                                                 ix, iy, iz)
+                                                 idwbuff(1), idwbuff(2), idwbuff(3))
```

Comment:
one day we will collapse these into `indices -> indices%x -> indices%x%beg`... just not yet

Reply:
I agree this could be improved when CCE is fixed. It's everywhere so we'll be reminded of it. I haven't thought about what the best solution is.

Reply:
can you open a GH issue for it (perhaps tag it as some future issue to worry about when Cray is more stable) then resolve this conversation?

Reply:
ref in #679